react/features/embed-meeting/components/Header.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import React from 'react';
+
+import { translate } from '../../base/i18n';
+import { Icon, IconClose } from '../../base/icons';
+
+type Props = {
+
+    /**
+     * The {@link ModalDialog} closing function.
+     */
+    onClose: Function,
+
+    /**
+     * Invoked to obtain translated strings.
+     */
+    t: Function
+};
+
+/**
+ * Custom header of the {@code EmbedMeetingDialog}.
+ *
+ * @returns {React$Element<any>}
+ */
+function Header({ onClose, t }: Props) {
```

Comment:
We seem to have this code in 3 places, with just the text changing. Please make it an action item to modify the base dialog to incorporate the new header after PR lands.